### ISSUE TITLE
Update came from the future

### ISSUE THREAD

**Issue by miketheman** [ewdurbin/amtrak#2]
I noticed that the "last updated" value for a train was in the future, and thought it odd - not sure how that can happen.
I tried to poke at the data responses from the API, but it was after the time had passed.
Is the API is returning incorrect data? Not sure how to catch/highlight that for further diagnosis.

**Comment by miketheman** [comment]
![Screenshot_20240610-102553.png](https://github.com/ewdurbin/amtrak/assets/529516/053befa6-8b10-47ea-98a4-5d23b8f71647)

**Comment by ewdurbin** [comment]
This value is what is recorded in their data's "LastValTS".

https://github.com/ewdurbin/amtrak/blob/c2450b35ac1913a060f3627b731c5b393dab875a/amtrak.py#L225
https://github.com/ewdurbin/amtrak/blob/c2450b35ac1913a060f3627b731c5b393dab875a/templates/train_partial.jinja2#L54

There isn't much that I can really account for other than that the train was pre departure. Looking at current status, I *am* able to reproduce for another pre departure train:

```diff
diff --git a/amtrak.py b/amtrak.py
index 73b5aef..8b719e5 100644
--- a/amtrak.py
+++ b/amtrak.py
@@ -230,6 +230,8 @@ def parse_trains(trains):
             .replace(microsecond=0)
             .astimezone(tz=TIMEZONES[cur_tz]),
         }
+        if parse_date(_train["properties"]["LastValTS"], cur_tz) > datetime.datetime.utcnow().replace(tzinfo=datetime.timezone.utc):
+          print(int(_train["properties"]["TrainNum"]), _train["properties"]["TrainState"], parse_date(_train["properties"]["LastValTS"], cur_tz))
         _trains[_train["properties"]["TrainNum"]].append(_data)
         if os.environ.get("STORE_DATA") is not None:
             Path(f"data/{_data['train_number']}/id").mkdir(parents=True, exist_ok=True)
```

```
refreshing trains...
241 Predeparture 2024-06-11 19:00:00-04:00
542 Predeparture 2024-06-11 15:50:00-07:00
719 Predeparture 2024-06-11 15:57:00-07:00
794 Predeparture 2024-06-11 15:57:00-07:00
618 Predeparture 2024-06-11 18:55:00-04:00
```

I'm hesitant to make a conditional here, seems like a fun lil quirk.

**Comment by ewdurbin** [comment]
Addressed in f4049d858b632ddf32b99391512019a8a4db4275, which will at least give folks who are curious some indicator as to why Amtrak is forecasting their "LastValTS" or has stopped providing updates.

<img width="354" alt="Screenshot 2024-06-11 at 6 58 47 PM" src="https://github.com/ewdurbin/amtrak/assets/1200832/d6bb7fb7-51d1-4102-af6e-e8d30ca517dd">
<img width="351" alt="Screenshot 2024-06-11 at 6 59 02 PM" src="https://github.com/ewdurbin/amtrak/assets/1200832/a1a61fe1-dcbf-439f-9257-81871086c67e">
